src/api/controllers/authController.js

```diff
@@ -62,4 +63,26 @@ export default class Auth {
       return sendError(status.NOT_FOUND, res, 'email', errors.unkownEmail);
     });
   }
+
+  /**
+   * Signout a user from the system
+   *
+   * @author: Frank Mutabazi
+   * @static
+   * @param {object} req - the request object
+   * @param {object} res - the result object
+   * @memberof signout
+   * @returns {object} - the response body
+   */
+  static async signout(req, res) {
+    const getTokenFromHeaders = req.headers.authorization;
```

Comment:
@MCFrank16 please, favour nouns for variable names and verbs for method/function names.

Instead of `getTokenFromHeaders`, what do you think would be a better name for this variable?

Reply:
I'll go with ``token`` only